web/src/utils/formatting.ts

```diff
@@ -64,21 +73,35 @@ const scalePower = function (maxPower: number | undefined) {
 
   if (value < 1) {
     return {
-      unit: PowerUnits.KILOWATTS,
+      unit: EnergyUnits.KILOWATT_HOURS,
       formattingFactor: 1e-3,
     };
   }
 
   if (value < 1e3) {
     return {
-      unit: PowerUnits.MEGAWATTS,
+      unit: EnergyUnits.MEGAWATT_HOURS,
       formattingFactor: 1,
     };
   }
 
+  if (value < 1e6) {
```

Comment:
does it make sense to turn this into a switch case now that the if satements have grown?
I feel like they are easier to read over repeated if statements.

Reply:
Good point, I've rewritten it to be cleaner by using a mapping :)